seastarx.hh

```diff
@@ -24,3 +31,19 @@ shared_ptr<T> make_shared(A&&... a);
 using namespace seastar;
 using seastar::shared_ptr;
 using seastar::make_shared;
+
+namespace seastar {
+
+template <typename Tag> class bool_class;
+template <typename T> class lazy_eval;
+template <typename T> class lazy_deref_wrapper;
+  
+}
+
+template <typename Tag> struct fmt::formatter<seastar::bool_class<Tag>> : fmt::ostream_formatter {};
+template <typename T> struct fmt::formatter<seastar::lazy_eval<T>> : fmt::ostream_formatter {};
+template <typename T> struct fmt::formatter<seastar::lazy_deref_wrapper<T>> : fmt::ostream_formatter {};
+template <> struct fmt::formatter<std::filesystem::path> : fmt::ostream_formatter {};
+template <typename T> struct fmt::formatter<seastar::shared_ptr<T>> : fmt::ostream_formatter {};
+template <typename T> struct fmt::formatter<seastar::lw_shared_ptr<T>> : fmt::ostream_formatter {};
```

Comment:
i'd prefer adding them in seastar. see https://github.com/scylladb/seastar/pull/1956

Reply:
and https://github.com/scylladb/seastar/pull/1957